src/supervisor/watchers/handlers/types.ts

```diff
@@ -101,6 +101,7 @@ export type V1ClusterList<T> = (
   pretty?: string,
   resourceVersion?: string,
   resourceVersionMatch?: string,
+  sendInitialEvents?: boolean,
```

Comment:
what is this for?

Reply:
Required by the latest Kubernetes API.
https://kubernetes.io/docs/reference/using-api/api-concepts/#streaming-lists

Reply:
what value do we set for this? or just undefined?

Reply:
It can be undefined and we do not set it -> so undefined.
e.g it's passed as a param in all the calls to `paginatedNamespacedList`